opam

```diff
@@ -5,9 +5,12 @@ homepage: "https://github.com/ocamllabs/higher"
 bug-reports: "https://github.com/ocamllabs/higher/issues"
 dev-repo: "https://github.com/ocamllabs/higher.git"
 license: "MIT"
-build: [
-  "ocaml" "pkg/build.ml" "native=%{ocaml-native}%"
-                         "native-dynlink=%{ocaml-native-dynlink}%"
+build:
+[[ "ocaml" "pkg/pkg.ml" "build"
+           "installer" "true" ]]
```

Comment:
This needs to read `"--installer" "true"` there was a bug in `topkg/pkg`'s OPAM file fixed by https://github.com/dbuenzli/carcass/commit/8d5fa4bfc46699481f2a6c3d32e0c9e7c79bbd35. After upgrade you can reinstall the carcasses with:
```
carcass setup --force carcass 
```
(This will overwrite existing carcasses but leave your flesh untouched)